Server/serverSlate.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/go-vgo/robotgo"
+)
+
+const KEY = "m"
+const KEY_PRESS_COUNT = 10
+
+func main() {
+	fmt.Println("Start server...")
+
+	ln, _ := net.Listen("tcp", "127.0.0.1:")
```

Comment:
move all the network communication and message parsing into a separate class (possibly in another file)